python/cuml/common/array.py

```diff
@@ -44,7 +44,8 @@ class CumlArray(Buffer):
     Parameters
     ----------
 
-    data : rmm.DeviceBuffer, array_like, int
+    data : rmm.DeviceBuffer, cudf.Buffer, array_like, int, bytes, bytearrar or
```

Comment:
`bytearray`

Reply:
What about replacing `bytes, bytearray or memoryview` with `bytes`-like. In Python this is used to mean anything that could be coerced to a `memoryview`.